modules/migration-installing-cam-operator-ocp-3.adoc

```diff
@@ -4,7 +4,7 @@
 [id="installing-cam-operator-ocp-3_{context}"]
 = Installing the Cluster Application Migration Operator on an {product-title} 3 source cluster
 
-You can install the Cluster Application Migration Operator manually on an {product-title} 3.7 (or later) source cluster, which does not support OLM.
+You can install the Cluster Application Migration Operator manually on an {product-title} 3 source cluster, which does not support OLM.
```

Comment:
s/, which does not support OLM./that does not support OLM./

(IBM p. 29)

Reply:
I will define OLM

No 3.x cluster supports OLM, so the non-restrictive wording ("which") is correct.

Reply:
@apinnick okay, but as someone that was a systems administrator in the past, it's confusing to me. Is it necessary to state that in the procedure?

It seems like the intent is to express that you can't use OLM to install Operators, really any Operator, because this is a 4.x feature. But does that matter here as a parenthetical?

Maybe it does, and that's fine. But it feels superfluous.

Reply:
In a way, it is superfluous. It's only an explanation for why we install CAM manually on 3.x, while we use OLM for 4.x. I could remove it.